### PR TITLE
Fix fix dsi studio

### PR DESCRIPTION
**Pull request by frheault**
This should make all the data in https://brain.labsolver.org/hcp_trk_atlas.html work!

### CODE REVIEW

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_fix_dsi_studio_trk.py [hunk 1]
```diff
@@ -127,11 +127,16 @@ def main():
...
     sft_fix = StatefulTractogram(sft.streamlines, args.in_dsi_fa,
-                                 Space.VOX)
+                                 Space.VOXMM)
+    sft_fix.to_vox()
+    sft_fix.streamlines._data -= get_axis_shift_vector(flip_axis)
+
     sft_flip = flip_sft(sft_fix, flip_axis)
 
+    sft_flip.to_rasmm()
+    sft_flip.streamlines._data -= [0.5, 0.5, -0.5]
```
Comment (arnaudbore) [inline scripts/scil_fix_dsi_studio_trk.py:138]: Will it work everytime ?

Comment (arnaudbore) [reply scripts/scil_fix_dsi_studio_trk.py:138]: Good enough and there is a warning at the beginning of the script imao it's good to go.

**Review by arnaudbore — APPROVED** [review]

**Pull request merged**